app/controllers/api/assignment_repos_controller.rb

```diff
@@ -9,7 +9,7 @@ class AssignmentReposController < API::ApplicationController
 
     def index
       repos = AssignmentRepo.where(assignment: @assignment).order(:id)
-      paginate json: repos
+      paginate json: repos, roster: @assignment.organization.roster
```

Comment:
Is there a test which covers this pagination change? Will it be surprising to folks when deployed?

Reply:
The `roster` kwarg here doesn't add anything to the JSON payload, it adds context that I can access in the serializer via `instance_options`. The alternative is to do something like `repo.assignment.organization.roster.roster_entries.find...` for every repo in the serializer 😬